src/components/People/index.tsx

```diff
@@ -139,7 +152,16 @@ export default function PeopleList(): JSX.Element {
     if (activeLocale) {
       refreshSearch();
     }
-  }, [debouncedSearchTerm, selectedOrganization, activeLocale]);
+  }, [debouncedSearchTerm, search, activeLocale]);
+
+  useEffect(() => {
+    const findTotalUsers = async () => {
+      const users = await search('', true);
```

Comment:
@constanzauanini here we fetch all users and skip tf contacts, to get total users

Reply:
Oh, ok, I missed that "true". Thanks!